test/controllers/account_controller_test.rb

```diff
@@ -421,7 +421,7 @@ def test_add_mugshot
   end
 
   def test_no_email_hooks
-    for type in [
+    [
```

Comment:
Makes Rubocop happier, but my real motive for changing this is that it make my text editor happier. (Folds behave better.)